controllers/provider-aws/pkg/webhook/controlplaneexposure/add.go

```diff
@@ -15,6 +15,7 @@
 package controlplaneexposure
```

Comment:
Can we now maybe find a better name than `controlplaneexposure`?

Reply:
As discussed, we don't have a good proposal now. If we come up with one, the renaming should be done in a separate PR.